src/StackExchange.Redis/Interfaces/IDatabaseAsync.cs

```diff
@@ -2477,6 +2478,10 @@ public interface IDatabaseAsync : IRedisAsync
         /// <remarks><seealso href="https://redis.io/commands/append"/></remarks>
         Task<long> StringAppendAsync(RedisKey key, RedisValue value, CommandFlags flags = CommandFlags.None);
 
+        /// <inheritdoc cref="StringBitCountAsync(RedisKey, long, long, StringIndexUnit, CommandFlags)" />
+        [Browsable(false), EditorBrowsable(EditorBrowsableState.Never)]
+        Task<long> StringBitCountAsync(RedisKey key, long start, long end, CommandFlags flags);
```

Comment:
I _think_ these are okay, but we should add a test suite to `OverloadCompat`	to ensure we're all good and haven't missed anything. I've merging in the `StringGet()` bits there as an example!

Reply:
I had these tests in `Strings.cs` and now moved it to `OverloadCompat.cs`.
@NickCraver Is it ok now?